src/openeo_test_suite/tests/general/payload_examples/post_result/post_result_1.json

```diff
@@ -0,0 +1,340 @@
+{
```

Comment:
This file is invalid, e.g. schemas, examples etc. Make sure only properties are provided when needed.